lucien-ios/ProfileViewController.swift

```diff
@@ -16,18 +16,56 @@ final class ProfileViewController: UIViewController {
     @IBOutlet private weak var name: UILabel!
     @IBOutlet private weak var emailTitle: UILabel!
     @IBOutlet private weak var email: UILabel!
-    @IBOutlet private weak var profileTitle: UILabel!
+
+    let loginViewModel = LoginViewModel()
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        configureNavigationController()
+        loginViewModel.getCurrentUser {
+            guard let currentUser = self.loginViewModel.currentUser else { return }
+            debugPrint("in debug print \(currentUser)")
+            self.name.text = currentUser.firstName + " " + currentUser.lastName
```

Comment:
The name and email should be properties in the view model that are set when the user data is fetched. Thus, `getCurrentUser` doesn't need a completion